app/assets/javascripts/metrics.js

```diff
@@ -156,3 +162,70 @@ function displayWeeklyBreakdown(where, rawDataSource) {
     
     chart.draw(processedData);
 }
+
+function thisYear() {
+    return parseInt(d3.select("#year").html());
+}
+
+function displayPerformanceLineChart(where, performanceData, volumeData) {
+    var xRange = [new Date(thisYear() - 1, 11, 29), new Date(thisYear() + 1, 0, 1)];
```

Comment:
Yeah...

According to ISO8601 the first week of the year can technically start in the previous year:

"It has 4 January in it. Hence the earliest possible dates are 29 December through 4 January, the latest 4 through 10 January."